README.rst

```diff
@@ -28,12 +28,31 @@ How can I contribute?
 What do I need to contribute?
 -----------------------------
 
-* A host with python 3.6+, gcc and the python development header files
-* tox
-* go
+* A host with `python` 3.6+ and `tox`
 * docker and/or podman+buildah
 * vagrant (optional, can be used to test FIPS mode and registered hosts)
 
+Test setup on openSUSE
+^^^^^^^^^^^^^^^^^^^^^^
+
+It is recommended to run BCI tests in its own virtual environment by doing the following:
+
+.. code-block:: shell-session
+    
+    $ sudo zypper in python3-virtualenv
+    $ virtualenv .bci_tester
+    $ source .bci_tester/bin/activate
+
+Once the virtual environment `.bci_tester` has been setup, you only need to run `source .bci_tester/bin/activate` again to activate it.
+
+To setup the test environment on openSUSE Leap 15.6 and Tumbleweed, run the following commands:
+
+.. code-block:: shell-session
+
+    $ sudo zypper -n in podman buildah docker git-core python3 python3-pip
```

Comment:
so is this supposed to be run inside the venv or outside? if it's inside the venv, you don't need to install python3-pip anymore, it is provided by the venv.

Reply:
True only if running in a venv. I phrased the previous block as optional so in cases where people run this outside a venv (for whatever reasons) it would still be needed.